Source/Fuse.Triggers/Trigger.uno

```diff
@@ -341,8 +341,11 @@ namespace Fuse.Triggers
 		{
 			if (!_isStarted)
 			{
-				if (!IsRootingStarted)
-					throw new Exception( "Trigger started prior to being rooted: " + this );
+				if (!IsRootingStarted){
+					debug_log("Warning: Trigger.uno - Trigger started prior to being rooted: " + this);
+					return;
+				}
```

Comment:
Hmm, this just hides the symptoms... Why do you want to do that rather than fixing the actual problem?

Reply:
I think we should call `Fuse.Diagnostics.UserWarning(message, object)` to report the warning instead.

https://github.com/fuse-open/fuselibs/search?q=UserWarning&unscoped_q=UserWarning

Reply:
Tried fixing the problem but it randomly occurs, don't know why things are not rooted at the time of triggering the animation, nor how to make sure that it is rooted for sure before triggering the animation.

Reply:
@kusma I don't think this have to be a fatal error, so reporting the problem via the Diagnostics API is maybe better than crashing the app.

Reply:
...switched it to Fuse.Diagnostics.UserError()